.nvmrc

```diff
@@ -1 +1 @@
-14.16.0
\ No newline at end of file
+14.18.3
```

Comment:
Should we use `16.14.2`? it's the LTS. If we do, it should probably be on another PR.

Reply:
good call, put in a ticket for it
https://github.com/Nonprofit-Exchange-Hub/web-app/issues/151
still going to bump up in this PR if that's ok